src/test/resources/data/lzo/test

```diff
@@ -0,0 +1,203 @@
+
+                                 Apache License
```

Comment:
Why are you compressing apache license?

Also, using binary file (with full 8-bit; call it `test.bin`) would offer better test coverage.

Reply:
It was handy and also used in the zstd tests. `AbstractTestCompression` has a full test suite. The reason to add these files is to test newer LZO than is generated by the Hadoop code. We're just trying to make sure the versions and flags work.